### PR TITLE
Add disable html support to monitor middleware.

### PR DESCRIPTION
**Pull request by efectn**
Add disable html support to monitor middleware.

### CODE REVIEW

**Comment by hi019** [comment]
I think APIOnly would be more descriptive name

**Comment by efectn** [comment]
> I think APIOnly would be more descriptive name

Yeah, should be better. I'll change variable name and fix tests.

**Comment by efectn** [comment]
@ReneWerner87 can you help me to fix test?

File: middleware/monitor/README.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ func main() {
 ```go
 // Config defines the config for middleware.
 type Config struct {
+	// To disable serving HTML, you can make true this option.
+	//
+	// Optional. Default: false
+	DisableHTML bool
```
Comment (ReneWerner87) [inline middleware/monitor/README.md:42]: don´t forget to change this to APIOnly

**Comment by ReneWerner87** [comment]
sure .... its a very strange bug in the tests

File: middleware/monitor/config.go [hunk 2]
```diff
@@ -4,14 +4,20 @@ import "github.com/gofiber/fiber/v2"
 
 // Config defines the config for middleware.
 type Config struct {
+	// To disable serving HTML, you can make true this option.
```
Comment (hi019) [inline middleware/monitor/config.go:7]:
```suggestion
	// Whether the service should expose only the monitoring API.
```

**Comment by efectn** [comment]
@ReneWerner87 can you check the PR?

**Comment by ReneWerner87** [comment]
Sure, tomorrow morning

**Review by ReneWerner87 — APPROVED** [review]

**Comment by ReneWerner87** [comment]
Sure

**Comment by ReneWerner87** [comment]
Please add this new configuration flag in our documentation repository

**Comment by ReneWerner87** [comment]
doc added https://github.com/gofiber/docs/commit/b67c5313f322a56b08874e17f292aef5b125f32a

**Pull request merged**